modules/sdk-core/src/bitgo/lightning/lightning.ts

```diff
@@ -91,13 +91,8 @@ export class Lightning implements ILightning {
 
   public async deposit(params: LightningDepositParams): Promise<DepositResponse> {
     const { amount } = params;
-    let { address } = params;
 
-    if (address === undefined) {
-      address = (await this.createDepositAddress()).address;
-    }
-
-    const res = await this.wallet.send({ amount, address });
+    const res = await this.wallet.send({ amount });
```

Comment:
We still need to specify an address when we send coins, we just want to make sure that we use `createDepositAddress` to get the address.